articles/sentinel/notebooks.md

```diff
@@ -42,47 +42,123 @@ We've also released some open-source Jupyter security tools in a package named [
-    > ![create workspace](./media/notebooks/sentinel-notebooks-azureml-create.png)
+    > ![create a new aml workspace](./media/notebooks/sentinel-azure-notebooks-aml-create.png)
 
-1. On the **Machine Learning** page, provide the following information, and then select **Review + create**.
+1. Provide the information for an Azure AML workspace. Then select **Next**.
 
     |Field|Description|
     |--|--|
     |Subscription|Select the Azure subscription that you want to use.|
-    |Resource group|Use an existing resource group in your subscription or enter a name to create a new resource group. A resource group holds related resources for an Azure solution. In this example, we use **AzureMLRG**.|
-    |Workspace name|Enter a unique name that identifies your workspace. In this example, we use **testworkspace1**. Names must be unique across the resource group. Use a name that's easy to recall and to differentiate from workspaces created by others.|
+    |Resource group|Use an existing resource group in your subscription or enter a name to create a new resource group. A resource group holds related resources for an Azure solution.|
+    |Workspace name|Enter a unique name that identifies your workspace. Names must be unique across the resource group. Use a name that's easy to recall and to differentiate from workspaces created by others.|
     |Region|Select the location closest to your users and the data resources to create your workspace.|
-    |Workspace edition|Select **Basic** as the workspace type in this example. The workspace type (Basic & Enterprise) determines the features to which you'll have access and pricing.|
+    |Storage account| A storage account is used as the default datastore for the workspace. You may create a new Azure Storage resource or select an existing one in your subscription.|
+    |KeyVault| A key vault is used to store secrets and other sensitive information that is needed by the workspace. You may create a new Azure Key Vault resource or select an existing one in your subscription.|
+    |Application insights| The workspace uses Azure Application Insights to store monitoring information about your deployed models. You may create a new Azure Application Insights resource or select an existing one in your subscription.|
+    |Container registry| A container registry is used to register docker images used in training and deployments. To minimize costs, a new Azure Container Registry resource is created only after you build your first image. Alternatively, you may choose to create the resource now or select an existing one in your subscription, or select **None** if you don't want to use any container registry.|
+
+    > [!div class="mx-imgBorder"]
+    > ![provide details for aml workspace](./media/notebooks/sentinel-notebooks-create-aml-details.png)
 
+1. On "Networking" page, you can connect to your workspace either publicly or privately using a private endpoint. Then select **review + Create**.
```

Comment:
```suggestion
1. On the **Networking** tab, you can choose either a public or a private endpoint through which to connect to your workspace. Then select **Review + create**.
```
It would be good to have an explanation here of the implications of choosing a public vs. a private endpoint, and why you might choose one or the other.